usaspending_api/data/COVID-19_download_readme.txt

```diff
@@ -0,0 +1,66 @@
+========ABOUT THESE FILES========
+
+This ZIP file was generated from the COVID-19 Profile page on USAspending.gov, located at https://www.usaspending.gov/#/disaster/covid-19.
+
+Data Element Definitions: A searchable Data Dictionary that defines every data element in the included files can be found here: https://www.usaspending.gov/#/download_center/data_dictionary. We have also included a copy in this download for convenience. The dictionary is updated periodically as the data model is improved or download headers change.
+
+Split Files: The # in all filenames defaults to 1; if the number of rows in a given file is large enough to warrant breaking it into multiple files, then additional files will be present and appended with 2, 3, etc. instead.
+
+-----------------
+
+FY20P07-Present_All_TAS_AccountBalances_2019-11-20_H23M06S44_1.csv
```

Comment:
Pretty trivial, but you'll never see a file with this date.

Reply:
This file is from Ross